### DISCUSSION TITLE
Is there a way to output pretty JSON?

### DISCUSSION THREAD

**Discussion by FaHuSchmidt** [Budibase/budibase#11916] [Help]
There are some situations where i just want to output a pretty formated JSON.
Is there any way?
I tried to use the JSON form field, but it always shows the escaped JSON. If i use JSON parse there will be something like this: "{{ JSONparse [cbbc9a6ac3ef94847a8c062244817c3e6].[data] }}"

**Comment by avivkri** [comment]
Hello @FaHuSchmidt,

You might want to use the same JSON form field, but use Javascript instead of Handlebars to assign its value - like
`return $("Action 6.Query result")` in Javascript instead of `{{ Action 6.Query result }}` in Handlebars

This should display the JSON as is without escaping.
